src/operations.js

```diff
@@ -1,4 +1,4 @@
 export const add = (left, right) => left + right
-export const minus = (left, right) => left - right
-export const times = (left, right) => left * right
-export const square = (number) => times(number, number)
+export const subtract = (left, right) => left - right
+export const multiply = (left, right) => left * right
+export const square = (number) => multiply(number, number)
```

Comment:
I've decided to rename the methods. `add` and `minus` aren't consistent. It's either plus, minus and times; the way we learn as children how to pronounce the +, - and x. 

Or addition, subtraction and multiplications, or as verbs add, subtract and multiply. This is the more math-y version.